src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -4377,7 +4384,9 @@ private bool RemoteDirectoryExist(System.Management.Automation.PowerShell ps, st
             if (op != null)
             {
                 if (op["Exists"] != null)
```

Comment:
Can this be refactored to `if (op?["Exists"] != null)` rather than a double-if? Just in the interest of tidying, but might be out of scope for this PR.

Reply:
True, will apply.